### PR TITLE
add bwidget recipe

### PR DESCRIPTION
**Pull request by fhoehle**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by fhoehle** [comment]
resolving: https://github.com/conda-forge/r-base-feedstock/issues/64

File: recipes/bwidget/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+build:
+  skip: True  # [win]
+  number: 0
+  script:
+    - cd src
+    # Remove this stuff so that lib is not polluted
+    - rm -Rf .fslckout ChangeLog tests demo BWman
+    - "[[ ! -d $PREFIX/lib/tcl8.6/bwidget ]] || exit 1"
+    - "mkdir -p $PREFIX/lib/tcl8.6/bwidget"
+    - "mv * $PREFIX/lib/tcl8.6/bwidget/"
```
Comment (scopatz) [inline recipes/bwidget/meta.yaml]: This is probably complex enough to be in a `build.sh` script

Comment (FelixHoehleQC) [reply recipes/bwidget/meta.yaml]: @scopatz moved to `build.sh`

Comment (fhoehle) [reply recipes/bwidget/meta.yaml]: moved to `build.sh`

**Comment by fhoehle** [comment]
@conda-forge/staged-recipes recipe ist ready further review or merging, thanks!

**Pull request merged**